source/content/guides/drupal-9-migration/05-drupal-9-to-pantheon.md

```diff
@@ -0,0 +1,377 @@
+
+1. Log in to your Pantheon account. If you don't have an account yet, [create one first](https://pantheon.io/register?docs) and familiarize yourself with the [User Dashboard](/guides/quickstart/user-dashboard) before you create a new site.
+
+1. Set up [SSH Keys](/ssh-keys) on your local computer and Pantheon account.
+
+1. Navigate to your User Dashboard and click the **Create New Site** button:
+
+  ...
+
+1. Click **Visit your Pantheon Site Dashboard**
+
+Now that you have a new site on Pantheon, you're ready to add the major components from your existing site: custom code, files, and the database.
+
+## Prepare the Local Environment
+
+<Partial file="drupal-9/prepare-local-environment.md" />
+
+Create a new folder to use while working on the migration. This folder will contain two subdirectories that you'll create in the next sections, one for the site on the former platform, and one for the Pantheon site.
+
+This doc uses the following aliases:
+
+- **Alias:** `SITE`
+- **Site Name:** `anita-drupal`
+- **Working folder** ?
+- **Old site folder** `FORMER-PLATFORM`
+- **Pantheon site folder** `PANTHEON-D9`
+
+### Create a Local Copy of the Old Site's Code
+
+1. Obtain a local copy of your old site's code.  Your **code** is all custom and contributed modules or plugins, themes, and libraries. The codebase should not include the `sites/default/files` directory, or any other static assets you do not want tracked by version control.
+
+1. Export the database and media files (`sites/default/files`) from the old platform, but do not add them or upload any files to Pantheon.
+
+### Retrieve a Local Copy of the Pantheon Site's Code
+
+1. From the **<span class="glyphicons glyphicons-wrench"></span> Dev** environment of the Site Dashboard, set the site's Development Mode to Git:
+
+  ![Git connection mode](../../../images/dashboard/connection-mode-git.png)
+
+1. Copy the `git clone` command for the site repository.
+
+  The command should look similar to the following:
+
+  ```bash
+  git clone ssh://codeserver.dev.{site-id}@codeserver.dev.{site-id}.drush.in:2222/~/repository.git
+  ```
+
+1. Run the `git clone` command inside your working folder
+
+<!--
+### Site Structure
+
+<Partial file="ic-upstream-structure.md" />
+-->
+
+## Add in the Custom and Contrib Code Needed to Run Your Site
+
+What makes your site code unique is your selection of contributed modules and themes, and any custom modules or themes your development team has created. These customizations need to be replicated in your new project structure.
+
+### Composer packages
+
+1. Copy your package list from the "requires" section of your site's composer.json and add it to the new site's composer.json. If your composer.json defines additional repositories or patches, copy those over too. Take care not to overwrite the `upstream-configuration` package & repository.
+
+  Note: if your old site has custom patches in its codebase, make sure to copy those over as well.
```

Comment:
is this where we should list components that Pantheon already includes that the user should remove from their list of things?

Reply:
I don't think so?  This relates to a "patches" section of composer.json people with more complex sites use.  I shared some screenshots via slack at one point, but we agreed it was getting too "into the weeds" for the document.

Again this is a question of balancing completeness for people with complex sites with simplicity for people who have simple sites.  My personal view was that the simple note is enough.  It's the type of thing where one might add "If you don't know what this means, then you don't have it and don't need to worry about it."

It's like instructions for replacing a car part, saying "If you have any custom aftermarket modifications to your car, now is the time you will need to re-attach those".  If you don't know what it means.. it's not you.